docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,55 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### `client_idle_timeout`
+
+This example configures a global idle timeout of **30 minutes**. After 30 minutes of inactivity, the session will be terminated.
+
+The `client_idle_timeout` in Teleport is a configurable setting that helps improve security by terminating inactive sessions after a specified period. It can be applied globally or per role, allowing for flexibility based on your organization's security policies.
+   The `client_idle_timeout` configuration ensures that SSH sessions, web application, kubectl, or database connection that remain inactive for a certain period of time
```

Comment:
Client idle timeout is definitely supported for desktop sessions.

I'm not sure that it works well with web applications, and I would be specific about kubectl and say `kubectl exec` sessions (as there's not really a concept of a session for generic kubectl commands).